### PR TITLE
Update documentation with missing analysis keywords

### LINKED ISSUE #6561: Undocumented keyword `INVERSION`
tldr;
Deprecated keys `USE_EE` & `USE_GE` both lead to `INVERSION`
We allow keywords `IES_INVERSION <int>` and `INVERSION <string>` which map to the same setting
Should be deprecate one of these, and document the other?
None of these are documented at all.
Both variants of inversion settings have been used according to azure logs, although less than 20 incidents over the last 3 months.

-------

FMU-docs does not seem to have anything on this keyword.

```
  def handle_special_key_set(
        self, var_name: str, value: Union[float, int, bool, str]
    ) -> None:
        if var_name in self.DEPRECATED_KEYS:
            logger.warning(
                f"The {var_name} key have been removed" f"use the INVERSION key instead"
            )
        elif var_name == "INVERSION":
            inversion_str_map = {
                "EXACT": 0,
                "SUBSPACE_EXACT_R": 1,
                "SUBSPACE_EE_R": 2,
                "SUBSPACE_RE": 3,
            }
```
Later this is mapped to IES_INVERSION. This setting is also found in the analysis module in the the UI.
Ref.
Screenshot 2023-11-10 at 14 46 45

### LINKED ISSUE #6570: Undocumented keyword `LOCALIZATION`
Using these keywords in configuration is supported;

```
ANALYSIS_SET_VAR STD_ENKF LOCALIZATION True
ANALYSIS_SET_VAR STD_ENKF LOCALIZATION_CORRELATION_THRESHOLD 0.42
```

But this is not documented at all in FMU-docs.

### LINKED ISSUE #6571: Undocumented keywords `IES_MIN|MAX|DEC_STEPLENGTH`
```
ANALYSIS_SELECT IES_ENKF
ANALYSIS_SET_VAR IES_ENKF IES_MAX_STEPLENGTH 0.12
ANALYSIS_SET_VAR IES_ENKF IES_MIN_STEPLENGTH 0.34
ANALYSIS_SET_VAR IES_ENKF IES_DEC_STEPLENGTH 1.4
```

Setting these variables in the configuration is supported but not documented at all.

### PR DESCRIPTION
**Pull request by andreas-el**
**Issue**
Resolves #6561
Resolves #6570
Resolves #6571

Screenshot 2023-11-16 at 16 44 05

Screenshot 2023-11-17 at 08 03 34

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

## Pre review checklist

- [x] Read through the code changes carefully after finishing work
- [x] Make sure tests pass locally (after every commit!)
- [x] Prepare changes in small commits for more convenient review (optional)
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Updated documentation
- [x] Ensured that unit tests are added for all new behavior (See 
    [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)),
    and changes to existing code have good test coverage.

## Pre merge checklist
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: docs/reference/configuration/keywords.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Keyword name                                                            Required
 :ref:`ECLBASE <eclbase>`                                                NO                                                                      Define a name for the ECLIPSE simulations.
 :ref:`STD_CUTOFF <std_cutoff>`                                          NO                                      1e-6                            Determines the threshold for ensemble variation in a measurement
 :ref:`ENKF_ALPHA <enkf_alpha>`                                          NO                                      3.0                             Parameter controlling outlier behaviour in EnKF algorithm
-:ref:`ENKF_TRUNCATION <enkf_truncation>`                                NO                                      0.99                            Cutoff used on singular value spectrum
+:ref:`ENKF_TRUNCATION <enkf_truncation>`                                NO                                      0.98                            Cutoff used on singular value spectrum
```
Comment (DanSava) [inline docs/reference/configuration/keywords.rst:35]: I guess you double-checked that `The default value of ENKF_TRUNCATION is 0.98` was not an old typo before changing  this to 0.98 :sweat_smile:

**Review by DanSava — APPROVED** [review]
LGTM

File: docs/reference/configuration/keywords.rst (hunk 1 above)
Comment (andreas-el) [reply docs/reference/configuration/keywords.rst:35]: I checked the codebase, but also now inspected the refactor that was done prior to this. They all have `0.98` as the default value.

**Review by DanSava — APPROVED** [review]
Looks nice good job! :man_dancing:

**Pull request merged**